config/locales/diaspora/en.yml

```diff
@@ -1091,18 +1091,23 @@ en:
 
   registrations:
     new:
+      title: "Join the community!"
+      subtitle: "You're about to create an account on %{podname}. One diaspora* account is enough to reach the whole network, you don't need to create a new account here if you already have one on another pod."
```

Comment:
A couple of minor alterations: "You're about to create an account on %{podname}. A single account is enough to reach the whole diaspora* network. You don't need to create a new account here if you already have one on another pod."

Reply:
Thanks, that's better.